From b193778aa7ee91e7677e2544353d3ce1844c4e63 Mon Sep 17 00:00:00 2001
From: Eric Engestrom <eric.engestrom@imgtec.com>
Date: Mon, 25 Apr 2016 14:54:42 +0100
Subject: [PATCH 074/134] trace-cmd record: Fix `const` placement

In `const char const * files[]`, both `const` refer to `char`.
Moving the second one after the `*` makes the pointer const as well,
which is probably what was intended.

Signed-off-by: Eric Engestrom <eric.engestrom@imgtec.com>
Signed-off-by: Steven Rostedt <rostedt@goodmis.org>
---
 trace-record.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/trace-record.c b/trace-record.c
index e4839943e8b1..9f220c9c6330 100644
--- a/trace-record.c
+++ b/trace-record.c
@@ -3606,7 +3606,7 @@ static void clear_func_filters(void)
 	struct buffer_instance *instance;
 	char *path;
 	int i;
-	const char const *files[] = { "set_ftrace_filter",
+	const char * const files[] = { "set_ftrace_filter",
 				      "set_ftrace_notrace",
 				      "set_graph_function",
 				      "set_graph_notrace",
-- 
2.5.5

